messaginginapp/src/main/java/io/customer/messaginginapp/ModuleMessagingInApp.kt

```diff
@@ -4,8 +4,8 @@ import android.app.Application
 import androidx.annotation.VisibleForTesting
 import io.customer.messaginginapp.di.gistProvider
 import io.customer.messaginginapp.hook.ModuleInAppHookProvider
-import io.customer.sdk.CustomerIO
 import io.customer.sdk.CustomerIOConfig
+import io.customer.sdk.android.CustomerIO
```

Comment:
Push and in-app will temporarily use old SDK instance, once these modules are decoupled, we can safely remove old instance and its usage.